### DISCUSSION TITLE
Question with /acb

### DISCUSSION THREAD

**Discussion by meta0ne** [GriefPrevention/GriefPrevention#882] [Q&A] [answered]
I have a question i'm trying to figure out, I'm using signs with CommandSign plugin to let players use to buy claimblocks. currently the executed command is /claimblocks 30. but i'd like to make discount prices rather that going with what grief prevention has set as the block price. I was looking to use /acb (placeholder for entity name) (quantity). I've tried to find a rememdy for this to work, but it justs wants a numeric value. any advice for a solution?

**Comment by TomLewis** [comment]
Youll have to give it a playername not entity name.

**Comment by meta0ne** [comment]
> Youll have to give it a playername not entity name.

So there is no way to make it to where it automatically pulls the person's name interacting with the command? no placeholders for such?

**Comment by Jikoo** [comment]
Could you clarify exactly what you're doing? I haven't used CommandSigns in years, and vague ideas (i.e. is `(placeholder for entity name)` the name of the executing player? Is it the name of the type of entity?) really confuse things.

Having looked at a little of the documentation for the version on Bukkit, you probably want something like this:
```
$100
/#acb <player> 100
```
If you're having trouble configuring CommandSigns, you should seek support from them.

**Comment by meta0ne** [comment, marked as answer]
Thank you! I got it working! when I tried to do this sign editing ingame, it gave errors. but when editing the sign configs it took the %player% placeholder. THANKS!!
